packages/components/src/internal/util/RouteLeave.tsx

```diff
@@ -90,9 +90,9 @@ export const useRouteLeave = (
     );
 
     useEffect(() => {
-        const currentRoute = routes[routes.length - 1];
+        const currentRoute = routes?.[routes.length - 1];
```

Comment:
When did/does this happen? Seems like a problem with configuration.

Reply:
This started happening for jest tests that I was writing for page components that wrapped `<BasePermissions>`, namely `PermissionManagementPage`.  BasePermissions uses `withRouteLeave` but PermissionManagementPage doesn't take any props so I didn't see a good way to configure the jest setup context. Thoughts?